packages/core/content-releases/admin/src/components/CMReleasesContainer.tsx

```diff
@@ -138,42 +140,67 @@ const AddActionToReleaseModal = ({
         {({ values, setFieldValue }) => {
           return (
             <Form>
-              <ModalBody>
-                <Flex direction="column" alignItems="stretch" gap={2}>
-                  <Box paddingBottom={6}>
-                    <SingleSelect
-                      required
-                      label={formatMessage({
-                        id: 'content-releases.content-manager-edit-view.add-to-release.select-label',
-                        defaultMessage: 'Select a release',
+              {releases?.length === 0 ? (
```

Comment:
I think ModalBody should be above this ternary, because the NoContent component needs to be in a ModalBody too

Reply:
If I add the ModalBody to wrap NoContent I have a box shadow and a margin outside the NoContent so I prefer to stay like this because it reflects the mockup